gemma-cli/src/main/java/ubic/gemma/apps/SingleCellDataDeleterCli.java

```diff
@@ -26,13 +26,14 @@ public class SingleCellDataDeleterCli extends ExpressionExperimentVectorsManipul
             DELETE_ALL_CELL_LEVEL_CHARACTERISTICS = "deleteClcs";
 
     @Autowired
-    private DataDeleterService dataDeleterService;
+    private ExpressionDataDeleterService expressionDataDeleterService;
 
     @Autowired
     private SingleCellExpressionExperimentService singleCellExpressionExperimentService;
 
     private String ctaIdentifier;
     private String clcIdentifier;
+    private boolean removeCellTypeFactorIfNecessary = true;
```

Comment:
The default should be to keep it, it can always be removed later on when loading a new preferred CTA.